### DISCUSSION TITLE
Not a bug - but could be done different

### DISCUSSION THREAD

**Discussion by HikeAndMap** [amanharwara/altus#167] [General]
Every time I start Altus - each tab shows at the top under the user icon:

Update available
Click to update WhatsApp >

then I click there, it just reloads the tab and that's it.

there's no update of course and after clicking it the message is gone..

No suggestions here what to do - except that I don't see the point of this message every single time I start Altus?

**Comment by amanharwara** [comment]
That is not an update related to Altus itself, actually. That is something that WhatsApp shows and I haven't been able to find a way to disable that.
